### ISSUE TITLE
How to filter by null relationship?

### ISSUE THREAD

**Issue by michaelgodshall** [miLibris/flask-rest-jsonapi#38]
How can I filter a relationship using a null value?  Flask-Restless uses`is_null` or `is_not_null`, but there doesn't seem to be a similar operator in Flask-Rest-JSONAPI.  Any recommendations on how to achieve the same result in Flask-Rest-JSONAPI?

**Comment by michaelgodshall** [comment]
I'm trying to use the `is_` operator, but passing different values like `'None'` or `false` doesn't work.

**Comment by akira-dev** [comment]
I released a new version 0.12.5 so now you can try this:

```
?filter=[{
    "not: {
        "name":"relation",
        "op":"any",
        "val": null
    }
}]
```

**Comment by AndreNalevaiko** [comment]
Has the fix occurred only on this operator?
I have the same problem in the filter:
 `{"name":"category_id","op":"in","val":[1,null]}`
